js/src/hooks/useAutoCreateAdsMCAccounts.js

```diff
@@ -105,19 +108,40 @@ const useAutoCreateAdsMCAccounts = ( createMCAccount ) => {
 		if ( which ) {
 			const handleCreateAccountCallback = async () => {
 				if ( which === CREATING_BOTH_ACCOUNTS ) {
+					clearDetailedErrorBySlot(
```

Comment:
@ankitrox Instead of having multiple `clearDetailedErrorBySlot` calls, let's update `clearDetailedErrorBySlot` to `clearDetailedErrorBySlots` where you can pass an array of slots you want cleared.